keyboards/cannonkeys/iron165/info.json

```diff
@@ -0,0 +1,7 @@
+{
+    "keyboard_name": "Iron165",
+    "url": "https://cannonkeys.com",
+    "maintainer": "awkannan",
+    "width": 16,
+    "height": 5
```

Comment:
should probably provide a default otherwise all your boards look like ortho on configurator :)